source/includes/fact-navigate-to-settings.rst

```diff
@@ -0,0 +1,15 @@
+To open the |compass| Settings Panel, click the gear icon on the left 
```

Comment:
Just a heads up, the font for the text on this include is different from the other steps on each page. To mitigate that, we might want to include the `.. step::` directive in here and format the include accordingly within the procedure directive. For example like: 

```
Compass Settings Panel 
~~~~~~~~~~~~~~~~~~

.. procedure:: 
   :style: normal 

   .. include:: <include file path> 

   .. step:: Toggle `Enable Devtools`
```

^ That should render like normal.